src/lib.rs

```diff
@@ -57,24 +55,27 @@ pub fn dummy_endpoint() -> String {
 ///
 /// # Examples
 ///
-/// ``` 
-/// extern crate tokio_core;
+/// ```rust
```

Comment:
i don't it is needed in rust doc comments

Reply:
Yep, old habit. Will remove.